pkg/cli/verifier/connectivity_egress_test.go

```diff
@@ -0,0 +1,206 @@
+package verifier
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+)
+
+func TestEgressRun(t *testing.T) {
+	testMeshName := "test"
+
+	testCases := []struct {
+		name            string
+		resources       []runtime.Object
+		trafficAttr     TrafficAttribute
+		srcPod          types.NamespacedName
+		srcConfigGetter ConfigGetter
+		expected        Result
+	}{
+		{
+			name: "http egress to httpbin.org on port 80",
+			resources: []runtime.Object{
+				&corev1.Pod{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "curl",
+						Namespace: "curl",
+					},
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name: constants.EnvoyContainerName,
+							},
+						},
+						ServiceAccountName: "curl",
+					},
+				},
+				&corev1.Pod{
```

Comment:
I don't think the httpbin pods are necessary here. These tests are verifying connectivity between meshed pods and external hosts

Reply:
Good observation. Forgot to remove this as I copy-pasted it from the more pod to pod tests.